evalml/pipelines/time_series_regression_pipeline.py

```diff
@@ -25,7 +25,7 @@ class TimeSeriesRegressionPipeline(TimeSeriesPipelineBase):
 
     Example:
         >>> pipeline = TimeSeriesRegressionPipeline(component_graph=["Simple Imputer", "Linear Regressor"],
-        ...                                                       parameters={"Linear Regressor": {"normalize": True},
+        ...                                                       parameters={"Simple Imputer": {"impute_strategy": "mean"},
```

Comment:
Why did we have to do this?

Reply:
Scikit-learn removed the `normalize` parameter from their linear models (it's been deprecated for a while before that, as well, you can see the note about it in old docs [here](https://scikit-learn.org/1.1/modules/generated/sklearn.linear_model.ElasticNet.html#sklearn.linear_model.ElasticNet))